test/test_biot_savart.py

```diff
@@ -23,6 +25,8 @@ def test_normal_use(centers, normals, radii, segment_numbers, fov_min, fov_max,
     assert(basis[:, :, :, 0].shape == fov_n)
 
     # Check expected values
-    actual = np.round(basis[:, :, :, 0].reshape(len(axis_answers)), decimals=7)
-    expected = np.round(np.asarray(axis_answers), decimals=7)
+    actual = np.round(basis[:, :, :, 0].reshape(len(axis_answers)), decimals=-2)
+    expected = np.round(np.asarray(axis_answers), decimals=-2)
+    print(actual)
+    print(expected)
```

Comment:
Once we have a debug flag I would print the output if we are in debug mode, for now I would remove.